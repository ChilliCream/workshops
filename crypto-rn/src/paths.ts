export const Paths = {
  Home: 'Home',
} as const;
