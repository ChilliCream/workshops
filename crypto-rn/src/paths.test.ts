import {Paths} from './paths';

describe('Paths', () => {
  it('should keep shape', () => {
    expect(Paths).toMatchSnapshot();
  });
});
