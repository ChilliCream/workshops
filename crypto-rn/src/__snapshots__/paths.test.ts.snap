// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paths should keep shape 1`] = `
{
  "Home": "Home",
}
`;
